code/modules/events/wizard/greentext.dm

```diff
@@ -84,19 +80,27 @@
 		return QDEL_HINT_LETMELIVE
```

Comment:
i beg you please kill this, i hate all the lavaland or wahtever crap that does this

Reply:
make the object indestructible or wahtever instead i dont care. just remove Destroy needing fucking `force` in order to actually do what it says

Reply:
ok done did it yea its completed yes